lib/paper_trail/type_serializers/postgres_array_serializer.rb

```diff
@@ -29,8 +29,7 @@ def deserialize(array)
       private
 
       def active_record_pre_502?
-        ::ActiveRecord::VERSION::MAJOR < 5 ||
-          (::ActiveRecord::VERSION::MINOR.zero? && ::ActiveRecord::VERSION::TINY < 2)
+        ::ActiveRecord.version < Gem::Version.new("5.0.2")
```

Comment:
Is `::ActiveRecord.gem_version` preferred here?

Reply:
Yes you are right, I confused both `gem_version` wraps the version string into a `Gem::Version` object for free